CHANGELOG.md

```diff
@@ -1,109 +1,197 @@
-maturity of the implementation. The final 1.0.0 release will be Aries Interop
-Profile 2.0-complete, and based on Python 3.7 or higher.
-
-### Breaking Changes
-
-As of release candidate 1.0.0-rc1, the only identified breaking change is the
-handling of "unrevealed attributes" during verification (see
-[\#1913](https://github.com/hyperledger/aries-cloudagent-python/pull/1913) for
-details). As few implementations of Aries Wallets support unrevealed attributes
-in an AnonCreds presentation, this is unlikely to impact any deployments.
-
-### Categorized List of Pull Requests
-
-In rc1, there are not a lot of new features, as the focus is on cleanup and
-optimization. The biggest is the inclusion with ACA-Py of a universal resolver
-interface, allowing an instance to have both local resolvers for some DID
-Methods and a call out to an external universal resolver for other DID Methods.
-Another significant feature is full support for Hyperledger Indy transaction
-endorsement for Authors and Endorsers. A new repo
+# 0.8.0-rc0
+
+## February 8, 2023
+
+0.8.0 is a breaking change that contains all updates since release 0.7.5. It
+extends the previously tagged `1.0.0-rc1` release because it is not clear when
+that release will be finalized. Many of the PRs in this release were previously
+included in the `1.0.0-rc1` release. The categorized list of PRs separates those
+that are new from those in the `1.0.0-rc1` release candidate.
+
+With this release, a new automated process publishes container images in the
+Hyperledger container image repository. New images for the release are
+automatically published by the GitHubAction Workflows: [publish.yml] and
+[publish-indy.yml]. The actions are triggered when a release is tagged, so no
+manual action is needed. The images are published in the [Hyperledger Package
+Repository under aries-cloudagent-python] and a link to the packages added to
+the repositories main page (under "Packages"). Additional information about the
+container image publication process can be found in the document [Container
+Images and Github Actions].
+
+There are not a lot of new Aries Framework features in this release, as the
+focus has been on cleanup and optimization. The biggest addition is the
+inclusion with ACA-Py of a universal resolver interface, allowing an instance to
+have both local resolvers for some DID Methods and a call out to an external
+universal resolver for other DID Methods. Another significant new capability is
+full support for Hyperledger Indy transaction endorsement for Authors and
+Endorsers. A new repo
 [aries-endorser-service](https://github.com/hyperledger/aries-endorser-service)
 has been created that is a pre-configured instance of ACA-Py for use as an
-Endorser service. While some work has been done on moving the default Python
-version beyond 3.6, more work is still to be done on that before the final
-v1.0.0 release.
+Endorser service.
```

Comment:
Did you mean the separate the paragraphs describing the new container images with this paragraph?  It breaks the flow a bit.

Reply:
I did mean to do that, but it does make it look terrible. The first part was intended to be a “new capability”, and the second part some extra detail.  I combined them as you will see.